### PR TITLE
Better possible Formables and Journal entries etc

### PR DESCRIPTION
**Pull request by Tziplays**
### Prussia:

add decision "Emser Depeche"
Req:
Nation is NGF
has researched Nationalism
is at war with france
year is before 1871

Benefit:
8% of north german Pops become loyalist

add Buff "retake whats rightful German" for 5 years
+15% trainig rate
+15% Army Off / Def
-10% military goods costs
+10% Kill rate
+10% Moral DMG

will fail if
is Germany / Germany exists

add Decision Turn your eye on the South Germans
Req:
NGF owns Alsace-Lorrain
NGF is not a unifcation candidate for Germany
is not Germany

Benefit:
NGF gains perma claims on: Bavaria, Franconia, Württemberg, Baden, Rheinland

will fail if: is Germany

### Scandinavia
Req: alles Vanilla ausser
Pan-Nationalismus -> Nationalismus
+Needs either Zealand State or Finland as Subject

Benefits:
-> add Norwegian, Finnish, Danisch, Estonian, Latvian and Lithuanian as prime culture
->perma claims on Ingria, Estonia, Latvia, Lithuania, East Prussia, West Prussia, Pommerania, Zealand, Jutland, Schleswig-Holstein, Mecklenburg

add perma Buff "Lion of the North"
+5% Off / Def Army + Navy
+5% Pop growth
-5% military goods costs
+10% Recovery Rate
+5% Manufactoring Income
+2% conscrition rate
Italy:

### Italy
Req: like Vanilla

Benefits:
->add Franco-Provencal, Albanian and Greece as prime culture
->give perma claims on Rhone, Provence, Peleponese, Attica, Crete, Thessaly, Macedonia, Albania, Corsica

add perma Buff "Resorgimento"
+5% Off / Def Army + Navy
+5% Tech Spread
+2% Pop growth
+5% state construction eff.
+1% conscription rate
### Austria
_created as a journal entry_
Empire of the Three Crowns
Req: Austria has Pan-Nationalism
Austria owns the following states:
->croatia, slovenia, Bosnia, Northern Serbia, Wallachia, Moldavia, Dobrudja

Benefits:
add perma claims on southern Serbia, Montenegro
add Serbien, Bosniak, Croatian and Romanian as primary culture
20% of these pops become loyalists
5% of south german/hungarian pops become radicals

add perma Buff Kingdom of the Three Crowns
+2% pop growth for accepted pops
+5% Authority
+5% state construction eff.
Spain/Portugal:

### Iberia
Req:
->Spain/Portugal has Nationalism (moved from Pan-nationalism)
->Spain/Portugal owns all States of Iberia either direct or via Dominon/Puppet

Benefits:
->Annexes All Puppets in Iberia
->Spain gains perma claims on Sicily, Sardinia, Corsica, al Rif, Fez, Marrakech, inner Morrocco
-> add Portuguese, Galician, Basque, Catalan as primary culture

Add perma Buff "a United Iberia"
+5% Off / Def Army + Navy
+5% Tech spread
+5% Taxation capacity
+3% birthrate
+5% Training rate
Portugal/Brazil:

### Reino Unido (Union with Brasil/Portugal)
_created as a journal entry_
Req:
->Either one has the other one as Dominion/Puppet
->Portugal / Brazil has Nationalism + Egalitarianism

Benefit:
Either one annexes the other one
->add portuguese, brazil, afrobrazil, paulista as primary culture (-das was die nation schon hatte)

add perma Buff "Reino Unido"
->+10% Off / Def Army / Navy
->+7% birthrate for acceptep pops
->+5% state construction eff
->+10% Plantations and Logging camps throughput
->+15% Uni throughput
-> -10% decree costs
Netherlands

### United Netherlands:
Req:
owns Flanders and Wallonia
Has nationalism

Benefits:
+25% of flemish / wallonian Pops become loyalists
gain perma claim on french lowlands, Cape Colony, Northern Cape, Eastern Cape, Vrystaat, Zululand, Transvaal

add perma Buff United Netherlands
+5% birthrate of accepted pops
+15% Naval Off / Def
+10% Uni throughput
+10% manufactoring throughput
+10% loyalists from SOL increase

### CODE REVIEW

File: src/common/country_formation/kmpmp_00_major_formables.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		}
+		OR = {
+			owns_entire_state_region = STATE_ZEALAND
+			OR = {
+				AND = {
+					owns_entire_state_region = STATE_SOUTHERN_FINLAND
+					owns_entire_state_region = STATE_ALAND
+					owns_entire_state_region = STATE_OSTROBOTHNIA
+				}
+				c:FIN = {
```
Comment (MasterSon-git) [inline src/common/country_formation/kmpmp_00_major_formables.txt:37]: c:FIN **?**=

File: src/common/country_formation/kmpmp_fomables.txt
Comment (MasterSon-git) [inline src/common/country_formation/kmpmp_fomables.txt:1]: file naming kmpmp_fo**r**mables.txt

File: src/common/country_formation/kmpmp_fomables.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+IBE = {
+	use_culture_states = yes
+
+	required_states_fraction = 0.9
+	
+	ai_will_do = { always = yes }
+
+	possible = {
```
Comment (MasterSon-git) [inline src/common/country_formation/kmpmp_fomables.txt:8]: müsste das possible nicht noch irgendwie auf kultur oder land eingeschränkt werden?

Comment (MasterSon-git) [reply src/common/country_formation/kmpmp_fomables.txt:8]: ebenso für UNL?

File: src/common/decisions/kmpmp_country_decisions.txt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+kmpmp_turn_your_eye_on_the_south = {
+	is_shown = {
+		NOT = {has_variable = took_decision_turn_eye_on_the_south}
+		OR = {
+			c:NGF ?= owner
+			c:PRU ?= owner
+		}
+		NOT = {exists = c:GER}
+	}
+	possible = {
```
Comment (MasterSon-git) [inline src/common/decisions/kmpmp_country_decisions.txt:40]: hier fehlt:
NGF is not a unifcation candidate for Germany

File: src/events/kmpmp_ibe_unification.txt
Comment (MasterSon-git) [inline src/events/kmpmp_ibe_unification.txt:1]: Um Events zu überschreiben muss du die komplette original Datei ersetzen also misc_unifications.txt kopieren
Und dann umbenennen so dass die das orignal überschreibt -> 00_kmpmp_misc_unifications.txt

File: src/events/kmpmp_ita_unification.txt
Comment (MasterSon-git) [inline src/events/kmpmp_ita_unification.txt:1]: hier auch komplette original Datei nehmen und umbenennen 00_kmpmp_italian_unification.txt

File: src/events/kmpmp_ita_unification.txt [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+		add_modifier = {
+			name = kmpmp_risorgimento
+		}
+		add_primary_culture = cu:francoprovencal
+		add_primary_culture = cu:albanian
+		add_primary_culture = cu:greek
+		give_claims_on_italy = yes
+		if = {
+			limit = {
+				NOT = { owns_entire_state_region = STATE_RHONE }
```
Comment (MasterSon-git) [inline src/events/kmpmp_ita_unification.txt:47]: sollte zwar so gehen wie du es gemacht hast, besser aber immer mit **s**:STATE_RHONE

File: src/events/kmpmp_sca_unification.txt
Comment (MasterSon-git) [inline src/events/kmpmp_sca_unification.txt:1]: das kommt dann auch nach 00_kmpmp_misc_formations.txt

File: src/events/kmpmp_unl_unification.txt
Comment (MasterSon-git) [inline src/events/kmpmp_unl_unification.txt:1]: das kommt dann auch nach 00_kmpmp_misc_formations.txt

File: src/localization/german/kmpmp_l_german.yml [hunk 5]
```diff
@@ -86,4 +86,21 @@
...
+ kmpmp_risorgimento:0 "Risorgimento buff"
+ kmpmp_kingdom_of_the_three_crowns:0 "Königreich der drei Kronen"
+ je_empire_of_the_three_crowns:0 "Königreich der drei Kronen"
+ je_empire_of_the_three_crowns_reason:0 "Der Balkan gehör uns!"
+  je_reino_unido_por:0 "Reino unido"
+ je_reino_unido_por_reason:0 "Vereint die Kronen"
+ je_reino_unido_brz:0 "Reino unido"
+ je_reino_unido_brz_reason:0 "Vereint die Kronen"
+ kmpmp_reino_unido:0 "Reino unido"
+ kmpmp_united_iberia:0 "Vereintes IBerien"
```
Comment (MasterSon-git) [inline src/localization/german/kmpmp_l_german.yml:105]: I**b**erien

**Review by MasterSon-git — CHANGES_REQUESTED** [review]
@Tziplays nen paar kleinigkeiten ;)

File: src/events/kmpmp_ita_unification.txt (hunk 4 above)
Comment (Tziplays) [reply src/events/kmpmp_ita_unification.txt:47]: hab das übernommen von vanilla
siehe event natural_borders

**Review by MasterSon-git — APPROVED** [review]

**Pull request merged**